packages/core/src/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Flex, Box } from 'pcln-design-system'
+
+const CustomBox = styled(Box)`
+  width: 100%;
+  border-radius: 2px;
+  margin-right: 4px;
+`
+
+const defaultStepColor = 'background.light'
+
+function ProgressBar({ steps, stepIndex, stepHeight }) {
+  return (
+    <Flex>
+      {steps.map((step, index) => {
+        const stepColor =
+          index < stepIndex ? steps[stepIndex - 1].color : defaultStepColor
```

Comment:
Does stepIndex start at 1 and that's why we're doing -1 here?

Reply:
yeah does that make sense to you? should we remove index from the name since that implies starting from 0, maybe `currentStep`?

Reply:
Yeah either that or just making it start from 0 would make sense to me.